src/app/system/system.service.ts

```diff
@@ -75,4 +75,28 @@ export class SystemService {
     return this.http.post('/codes', code);
   }
 
+  /**
+   * @returns {Observable<any>} Configurations data.
+   */
+  getConfigurations(): Observable<any> {
+    return this.http.get('/configurations');
+  }
+
+  /**
+   * @param {string} configurationId which needs to be fetched.
+   * @returns {Observable<any>} Configurations.
+   */
+  getConfiguration(configurationId: string): Observable<any> {
+    return this.http.get(`/configurations/${configurationId}`);
+  }
+
+  /**
+   * @param {string} configurationId Configuration ID of the configuration which needs to be updated.
+   * @param {any} configuration Changes to be done in configuration.
```

Comment:
```suggestion
   * @param {any} configuration Configuration to be updated.
```

Reply:
Here we are only passing the changes, are you sure to do this change?

Reply:
Yes.